packages/tezos-context-hash-irmin/tezos-context-hash-irmin.1.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Irmin implementation of the Tezos context hash specification"
+description: "Irmin implementation of the Tezos context hash specification"
+maintainer: ["Tarides <contact@tarides.com>"]
+authors: [
+  "Clément Pascutto <clement@pascutto.fr>"
+  "Mattias Roux <mattias@tarides.com>"
+  "Thomas Gazagnaire <thomas@tarides.com>"
+]
+license: "MIT"
+homepage: "https://github.com/tarides/tezos-context-hash"
+bug-reports: "https://github.com/tarides/tezos-context-hash/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "tezos-context-hash"
```

Comment:
is this going to be a package that you want to synchronize/release together with tezos-contex-hash-irmin or should they be more dislocated? (i.e. do you want `{= version}` or do you prefer to update previous constraints manually later?)

Reply:
I think it's fine if they are released separately - tezos-context-hash doesn't have to follow irmin releases whereas tezos-context-hash-irmin has.